### ISSUE TITLE
High severity vulnerability (axios)

### ISSUE THREAD

**Issue by alexkuc** [bundlewatch/bundlewatch#313]
Cloning repository reveals:

```
found 1 high severity vulnerability
```

Investigating further `npm audit`:

```

                       === npm audit security report ===

# Run  npm install axios@0.21.1  to resolve 1 vulnerability
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Server-Side Request Forgery                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ axios                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ axios                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ axios                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1594                            │
└───────────────┴──────────────────────────────────────────────────────────────┘


```

Please bump dependency for axios to resolve this (I think I can provide a PR if necessary)

**Comment by XhmikosR** [comment]
This has [a PR open](#310) with CI passing, but I'm not sure if it's a non-breaking change plus we need a new release made after the PR is merged.

**Comment by XhmikosR** [comment]
This was fixed in 0.3.2
